apps/etterlatte-behandling/src/main/kotlin/sak/SakService.kt

```diff
@@ -139,6 +142,14 @@ class SakServiceImpl(
         return dao.oppdaterAdresseBeskyttelse(sakId, adressebeskyttelseGradering)
     }
 
+    override fun hentSakMedUtenlandstilknytning(fnr: String): SakUtenlandstilknytning {
+        val finnSaker = dao.finnSaker(fnr)
+        if (finnSaker.size > 1) {
+            throw IllegalStateException("Person ${fnr.maskerFnr()} med sakider ${finnSaker.map { it.id }} må håndteres")
+        }
+        return dao.hentUtenlandstilknytningForSak(finnSaker[0].id)!! // Obs, ved begge ytelsene vil dette bli random
```

Comment:
Det vil vel ikke bli random, siden det kastes exception over hvis det er mer enn én?

Og hvis det er akkurat 1 som er forventet, hadde det vært ryddigere med noe slikt i stedet for en if-sjekk?
```kotlin
check(finnSaker.size == 1) {
    "Person ${fnr.maskerFnr()} med sakider ${finnSaker.map { it.id }} må håndteres"
}
```

Reply:
Jepp! Endret litt struktur med egne exceptions her https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/2614/commits/eb5ed7dfb91b6b51a982d785b0064100192a9c70